### PR TITLE
Create mobile portfolio

### PR DESCRIPTION
**Pull request by sevinchek**
## My Mobile Portfolio

In this branch I:
- Set up a Linter
- Added the header
- Added the headline section
- Added a README template
- Added an App Screenshot

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <h4>Kevin Sevinche</h4>
```
Comment (Davidosky007) [inline index.html:13]: you can make this in an anchor tag as this should serve as your portfolio logo as your design states

File: style.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navbar > h4 {
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (Davidosky007) [inline style.css:20]: i think you can make the font family for this section "Roboto" as stated in your design

File: style.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  display: flex;
+  flex-direction: column;
+  background-color: #1c1a19;
+}
+
+h1 {
+  color: #ff6b00;
+  font-size: 45px;
+  font-weight: 500;
+  font-family: monospace;
```
Comment (Davidosky007) [inline style.css:43]: you should use this "font-family: Crete Round" instead as stated in the design

File: style.css [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  margin-bottom: -11px;
+}
+
+h3 {
+  color: #fff5e1;
+  font-size: 21px;
+  font-family: Helvetica, sans-serif;
```
Comment (Davidosky007) [inline style.css:50]: you can make this font family Roboto as it is in the design template

File: style.css [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+  font-weight: 500;
+  margin-bottom: 0;
+}
+
+p {
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (Davidosky007) [inline style.css:57]: you can make this font family Poppins as stated in the design

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @sevinchek, 

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* I left you some comments under this review, please 🙏 check them and make the necessary changes 👍 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 6]
```diff
@@ -1 +1,21 @@
...
+- HTML & CSS
+
+## Authors
+
+👤 **Kevin**
+
+- GitHub: [@githubhandle](https://github.com/sevinchek)
+- Twitter: [@twitterhandle](https://twitter.com/sevinchek)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/sevinchek)
+
```
Comment (MahmoudBakr23) [inline README.md:18]: - [ ] **OPTIONAL**
Good job in the README, but there is always room for improvement.
Kindly check the [template](https://github.com/microverseinc/readme-template) and see what else you could have added to the README to make it more professional.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100&family=Roboto:wght@100&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <a href="" class="logo">Kevin Sevinche</a>
+            <a href="" class="menu"><img class="img-menu" src="https://igorashs.github.io/embedding-images-and-video/images/menu.svg" alt=""></a>
```
Comment (MahmoudBakr23) [inline index.html:17]: - [ ] **OPTIONAL**
Please note that this element is a button that exists to do something (which is to open the nav menu on the mobile screen) so, and according to the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp), it doesn't make sense to wrap it inside a (`<a>` tag) right? tell me what you think!
_I'm sorry that you didn't get notified of this specific point in the first review. But, I highly recommend adjusting it_

File: style.css [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+  height: 30px;
+  width: 30px;
+}
+
+section {
+  padding: 120px 24px;
+  display: flex;
+  flex-direction: column;
+  background-color: #1c1a19;
+}
```
Comment (MahmoudBakr23) [inline style.css:45]: - [ ] **OPTIONAL**
Fantastic work for styling the background of this section the hard way by creating those bottom three divs and styling them with CSS. But, don't you think it'd be way too better if you just made the background of the section as an image using the `background image` CSS property! Tell me what you think. 😃
If you do, it'll save you from all the extra CSS code and will solve the extra white space issue. Please look at the image down below! Then, compare it with the [original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
Screenshot

_I'm sorry that you didn't get notified of this specific point in the first review. But, I highly recommend adjusting it_

File: style.css [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+  color: #ff6b00;
+  font-size: 45px;
+  font-weight: 500;
+  font-family: 'Crete Round', sans-serif;
+  margin-bottom: -11px;
+}
+
+h3 {
+  color: #fff5e1;
+  font-size: 21px;
```
Comment (MahmoudBakr23) [inline style.css:57]: - [ ] Kindly follow the exact font size, weight, and colors as the given design. For this `<h3>` tag, please adjust the font size to be 24 px as the given design.

File: style.css [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  margin-bottom: 0;
+}
+
+p {
+  color: white;
+  font-family: Poppins, sans-serif;
+}
```
Comment (MahmoudBakr23) [inline style.css:66]: - [ ] Kindly follow the exact font size, weight, and colors as the given design. For this `<p>` tag as well. It still misses some styling to match the original design.

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @sevinchek 😃, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

_Check the **OPTIONAL** comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shaher-11 — APPROVED** [review]
Hello @sevinchek,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 

## Optional Suggestions 💡
_The following point is optional to implement however it's highly recommended!_:blush:

- Add Project Setup instructions in the Readme file.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**